### ISSUE TITLE
Indentation in Sass Maps

### ISSUE THREAD

**Issue by alan-agius4** [stylelint/stylelint#1458]
"A bug where the indentation rule warning is being firing when it is not suppose to in nested sass maps". The fix for https://github.com/stylelint/stylelint/issues/1406 seems to cause more issues even when not nesting maps. (which before worked fine)

> Which rule, if any, is this issue related to?

e.g. `indentation`

> What CSS is needed to reproduce this issue?

e.g.

```css
$colors: (
	color: #fff;
);

$colors-secondary: (
	primary: (
		base: #000,
		contrast: #fff
	)
);
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
"indentation": [
	"tab",
	{
		"indentClosingBrace": false,
		"indentInsideParens": "once" 
	}
],
}
```

> Which version of stylelint are you using?

`6.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"gulp"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

"Yes, it's related to SCSS maps.

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
2:2     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
3:3     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
4:3     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
6:2     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
7:2     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
8:2     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
9:2     ‼  Expected indentation of 0 tabs (indentation) [stylelint]
10:2    ‼  Expected indentation of 0 tabs (indentation) [stylelint]
```

**Comment by ntwb** [comment]
Related: #1149, #1451

**Comment by davidtheclark** [comment]
Fix committed in #1432. Does that not work? Why is this re-opened?

**Comment by jeddy3** [comment]
It's a new issue that, I think, @alan-agius4 accidentally closed and then reopened. He's using `6.6.0`.

**Comment by alan-agius4** [comment]
Yes in need, i closed it by mistake

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/pull/1493

**Comment by alan-agius4** [comment]
Was this released? As I cannot see anything in the changelog

Thanks

**Comment by jeddy3** [comment]
[`6.7.0`](https://github.com/stylelint/stylelint/blob/master/CHANGELOG.md#670) - "Fixed: `indentation` better understands nested parentheticals that aren't just Sass maps and lists."

**Comment by Arcanemagus** [comment]
https://github.com/stylelint/stylelint/commit/50cb15baeeeb8cf922daa9ea492bce740a73fcec shows that it is in v6.7.0 and up.
